packages/vats/src/lib-board.js

```diff
@@ -340,6 +341,11 @@ export const prepareBoardKit = baggage => {
           const readonly = makeReadonlyMarshaller(this.state);
           return JSON.stringify(readonly.serialize(val));
         },
+        parseAndDecode(str) {
+          const data = JSON.parse(str);
+          const readonly = makeReadonlyMarshaller(this.state);
+          return readonly.unserialize(data);
```

Comment:
this should be `fromCapData`. the Endo sync is imminent and will save some churn to wait for it:  https://github.com/Agoric/agoric-sdk/pull/7273/files#diff-298ac87f534f4fb60e42f478a8a866d0eb2f4e4853f6c7cb12c7f99c92594b2eR22

Reply:
> will save some churn to wait for it

ok